frontend/components/draft/sortable-lesson.tsx

```
@@ -3,7 +3,7 @@ import Link from "next/link";
 import { useRouter } from "next/navigation";
 import { useSortable } from "@dnd-kit/sortable";
 import { CSS } from "@dnd-kit/utilities";
-import { cn } from "@/lib/utils";
+import { cn, stripHtmlTags } from "@/lib/utils";
```

ESLint

[error] 'stripHtmlTags' is defined but never used.

(@typescript-eslint/no-unused-vars)